### PR TITLE
Class method

### PR DESCRIPTION
**Pull request by Kaiserabbas**
Hi Everyone,
- Used JavaScript classes instead of objects and arrays.
- Used objects and arrays inside the class.
- Created class methods to add and remove books.
- Maintained all of the original functionality of the application.
- Added CSS styles to the application to make it match the provided design.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -6,25 +6,33 @@ const books = JSON.parse(localStorage.getItem('books')) || [];
 function saveBooks() {
   localStorage.setItem('books', JSON.stringify(books));
 }
+// defining a class for form inputs
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
```
Comment (levy002) [inline script.js]: - [ ] Kindly make sure your `Book` class has `add` and `remove` methods in order to meet the project requirements.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Kaiserabbas    , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kaiserabbas** [comment]
Book class is included with add and remove methods to meet the project requirements.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @Kaiserabbas and @Titus-Kiplagat,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**